src/lib/table/createTableData.ts

```diff
@@ -0,0 +1,86 @@
+import type { ChartSeries } from '@/lib/charts/types/ChartSeries'
+import { Series } from '@/lib/timeseries/timeSeries'
+import { uniqWith } from 'lodash'
+
+export function createTableData(
+  chartSeriesArray: ChartSeries[] | undefined,
+  seriesRecord: Record<string, Series>,
+  seriesIds: string[],
+): Record<string, unknown>[] {
+  if (chartSeriesArray === undefined) return []
+  const dateTimes = createDateTimes(chartSeriesArray, seriesRecord)
+
+  const chartSeries = uniqWith(
+    chartSeriesArray.filter((s) => seriesIds.includes(s.id)),
+    (a, b) => {
+      return a.id === b.id
+    },
+  )
+  const p = Array(seriesIds.length).fill(0)
```

Comment:
provide a meaningfull name. p is bit confusing

Reply:
Renamed to `pointers`